### DISCUSSION TITLE
fixed hudl1 but playstore won't play

### DISCUSSION THREAD

**Discussion by chamomile** [remay/tmf-hudl#47] [Help]
I managed to complete the flash image update and the hudl1 now seems to work 'ok' - up to a point. I can get online now  but Playstore  doesn't work properly and  the chrome browser seems broken. I notice the android version is 4.2.2 (not sure what it was before I stupidly did the  factory reset that  broke the hudl1 in the first place.) Does anyone know how to mend or update these basic apps? 
Can a later version of android than 4.2.2 be installed? (if that's the problem,,,) or is it a lost cause?

**Comment by remay** [comment]
Yes - The Google Play Store no longer works with Android versions less than 5.  Hudl's latest firmware is v4.2.2.

See https://github.com/remay/tmf-hudl/issues/43 for more information and a potential way forwards.   I'm sorry that I haven't (yet) had time to get to looking at whether a better solution may be possible.
